src/metatrain/cli/train.py

```diff
@@ -308,6 +308,31 @@ def train_model(
             )
             validation_datasets.append(validation_dataset)
 
+    ###########################
+    # PRINT DATASET STATS #####
+    ###########################
+
+    for i, train_dataset in enumerate(train_datasets):
```

Comment:
We can also print this directlty below we load the datasets instead of here. But I don't mind much.